src/components/ingredient/recipeDetail/index.module.scss

```
@@ -0,0 +1,65 @@
+.recipe_title,
+.recipe {
+  background-color: #fff;
+  box-shadow: 0 0px 20px rgba(0, 0, 0, 0.2);
+  padding: 15px;
+}
+.recipe_title {
+  margin-bottom: 20px;
+  display: flex;
+  justify-content: left;
+  .title_container {
+    margin-left: 10px;
+  }
+  .recipe_image {
+    width: 100px;
+    height: 100px;
+  }
+  .title {
+    font-size: 24px;
+    font-weight: normal;
+    margin: 0;
+  }
+  .user {
+    display: flex;
+    margin-bottom: 20px;
+    margin-top: 10px;
+  }
+  .google_logo {
+    display: flex;
+
+    .google_icon {
+      margin-left: 120px;
+    }
+  }
+  .user_name {
+    font-size: 15px;
+    margin: 0;
+  }
+  .icon {
+    margin-right: 10px;
+    margin-left: 4px;
+  }
+}
+ul {
+  padding: 0;
+  justify-content: space-between;
+  margin-bottom: 50px;
+  list-style: none;
+  li {
+    display: flex;
+    justify-content: space-between;
+    padding: 5px 0;
+    border-bottom: 1px lightgray solid;
+  }
+}

```

要素名をselectorにする場合はこうしてみるとより安全なコードになります！
```suggestion
> ul {
  padding: 0;
  justify-content: space-between;
  margin-bottom: 50px;
  list-style: none;
  > li {
    display: flex;
    justify-content: space-between;
    padding: 5px 0;
    border-bottom: 1px lightgray solid;
  }
}

```